docs/operations/installing_operator_into_cluster_de.md

```diff
@@ -14,7 +14,7 @@ Die Installation von GitHub erfordert die Installations-YAML, die alle benötigt
 
```

Comment:
This documentation is very old and should be changed according the helm installation method.